lars-reimann/api-editor
# feat: load usage data into editor

Closes #494.

### Summary of Changes

Add way to upload usage data into the editor.

### Screenshots (if necessary)

#### Import Dropdown

![image](https://user-images.githubusercontent.com/2501322/169296840-a7f25bc5-0791-4756-b79b-21fff31b8beb.png)

#### Import Dialog

![image](https://user-images.githubusercontent.com/2501322/169296915-e1a8f645-5489-4dd9-8ddb-6ff1cd34b6ca.png)

### Testing Instructions

1. Click on the import dropdown in the menu bar
2. Click on the "Usages" entry
3. Click on the dropzone
4. Select a usage file
5. Click submit
6. Ensure no errors are thrown
